docs/rancher/cloud-provider.md

```diff
@@ -191,6 +191,12 @@ RKE/K3s upgrade cloud provider via the Rancher UI, as follows:
 3. Select **Version**. 
 4. Click **Next > Update**.
 
+:::note
+
+If the RKE2/RKE/K3s cluster consists of a single node, the upgrade might be stalled because the new cloud-provider Pod is stuck in a pending state. The reason behind this is that there is a section describing the rolling update strategy in the harvester-cloud-provider Deployment. The default value conflicts with the `podAntiAffinity` configuration in single-node cluster circumstances; hence, the upgrade is stalled. For more in-depth analysis, please refer to the [relevant GitHub issue comment](https://github.com/harvester/harvester/issues/5348#issuecomment-2055453709). To untie the stalemate, please manually delete the old harvester-cloud-provider Pod. You might need to do this multiple times until the new Pod can be successfully scheduled.
+
+:::
```

Comment:
```suggestion
:::info

The upgrade process for a [single-node guest cluster](../advanced/singlenodeclusters) may stall when the new `cloud-provider` pod is stuck in the *Pending* state. This issue is caused by a section in the `harvester-cloud-provider` deployment that describes the rolling update strategy. Specifically, the default value conflicts with the `podAntiAffinity` configuration in single-node clusters. 

For more information, see [this GitHub issue comment](https://github.com/harvester/harvester/issues/5348#issuecomment-2055453709). To address the issue, manually delete the old `harvester-cloud-provider` pod. You might need to do this multiple times until the new pod can be successfully scheduled.

:::
```

Reply:
I removed "RKE2/RKE/K3s" because the term [guest cluster](https://docs.harvesterhci.io/v1.3/glossary#guest-cluster--guest-kubernetes-cluster) is already defined using the supported distros/OSs.

Reply:
Updated, thank you!